geopy/geocoders/arcgis.py

```diff
@@ -218,19 +245,36 @@ def _refresh_authentication_token(self):
             raise GeocoderAuthenticationFailure(
                 'Too many retries for auth: %s' % self.retry
             )
-        token_request_arguments = {
-            'username': self.username,
-            'password': self.password,
-            'expiration': self.token_lifetime,
-            'f': 'json'
-        }
+
+        if self.username:            
+            auth_url = self.auth_api
+            token_request_arguments = {
+                'username': self.username,
+                'password': self.password,
+                'expiration': self.token_lifetime,
+                'f': 'json'
+            }
+        elif self.client_id:
+            auth_url = self.oauth_api
+            token_request_arguments = {
+                'client_id': self.client_id,
+                'client_secret': self.client_secret,
+                # User/pass method takes token lifetime in seconds, 
+                # OAuth method takes token lifetime in minutes
+                'expiration': self.token_lifetime / 60, 
```

Comment:
Shouldn't it be rounded to int? In Python 3 division result is `float`, but in Python 2 it's `int`.